web-jehlomat/src/types.ts

```diff
@@ -19,3 +19,13 @@ export interface IUser {
     isAdmin: boolean;
 }
 
+export interface IUserEdited {
```

Comment:
Is it really necessary to have special interface for user update? Can't we merge it with already existing User type? I mean, if I set all the non-required props to undefined I should get undefined values also for read operation.

Reply:
API accept different payload. So I prefer to keep it separated

Reply:
@fechtner please have a look, probably Swagger is wrongly cpecified.